www/js/diary/cards/UntrackedTimeCard.tsx

```diff
@@ -9,48 +9,41 @@
 
 import React from "react";
 import { View, StyleSheet } from 'react-native';
-import { Divider, IconButton, Text } from 'react-native-paper';
-import { object } from "prop-types";
+import { Divider, Text } from 'react-native-paper';
 import { getTheme } from "../../appTheme";
 import { useTranslation } from "react-i18next";
 import { DiaryCard, cardStyles } from "./DiaryCard";
 import { useAddressNames } from "../addressNamesHelper";
+import { Icon } from "../../components/Icon";
+import useDerivedProperties from "../useDerivedProperties";
+import StartEndLocations from "../StartEndLocations";
 
-const UntrackedTimeCard = ({ triplike }) => {
+type Props = { triplike: {[key: string]: any}};
+const UntrackedTimeCard = ({ triplike }: Props) => {
   const { t } = useTranslation();
+  const { displayStartTime, displayEndTime, displayDate } = useDerivedProperties(triplike);
   const [ triplikeStartDisplayName, triplikeEndDisplayName ] = useAddressNames(triplike);
 
   const flavoredTheme = getTheme('untracked');
 
   return (
     <DiaryCard timelineEntry={triplike} flavoredTheme={flavoredTheme}>
-      <View style={[cardStyles.cardContent, {marginVertical: 12}]}>
-        <View>{/*  date and distance */}
+      <View style={[cardStyles.cardContent, {marginVertical: 12}]} focusable={true}
+          accessibilityLabel={`Untracked time from ${displayStartTime} to ${displayEndTime}`}>
```

Comment:
Do these also need to be internationalized? I assume so!? Because if you are a spanish speaking user using a screenreader, presumably you want to hear it read out in Spanish. Unless the screenreader does some kind of translation by itself?

Reply:
I was wondering that, but figured that English labels were at least a good starting place.
Let's ask UW